{
	"document": {
		"aggregate_severity": {
			"namespace": "https://www.suse.com/support/security/rating/",
			"text": "moderate"
		},
		"category": "csaf_security_advisory",
		"csaf_version": "2.0",
		"distribution": {
			"text": "Copyright 2024 SUSE LLC. All rights reserved.",
			"tlp": {
				"label": "WHITE",
				"url": "https://www.first.org/tlp/"
			}
		},
		"lang": "en",
		"notes": [
			{
				"category": "summary",
				"text": "Security update for python-PyKMIP",
				"title": "Title of the patch"
			},
			{
				"category": "description",
				"text": "This update for python-PyKMIP fixes the following issues:\n\nSecurity issue fixed:\n\n- CVE-2018-1000872: Fixed a denial-of-service vulnerability which was caused by exhausting \n  the available sockets. To mitigate the issue server socket timeout was decreased (bsc#1120767).\n",
				"title": "Description of the patch"
			},
			{
				"category": "details",
				"text": "HPE-Helion-OpenStack-8-2019-391,SUSE-2019-391,SUSE-OpenStack-Cloud-8-2019-391,SUSE-OpenStack-Cloud-Crowbar-8-2019-391",
				"title": "Patchnames"
			},
			{
				"category": "legal_disclaimer",
				"text": "CSAF 2.0 data is provided by SUSE under the Creative Commons License 4.0 with Attribution (CC-BY-4.0).",
				"title": "Terms of use"
			}
		],
		"publisher": {
			"category": "vendor",
			"contact_details": "https://www.suse.com/support/security/contact/",
			"name": "SUSE Product Security Team",
			"namespace": "https://www.suse.com/"
		},
		"references": [
			{
				"category": "external",
				"summary": "SUSE ratings",
				"url": "https://www.suse.com/support/security/rating/"
			},
			{
				"category": "self",
				"summary": "URL of this CSAF notice",
				"url": "https://ftp.suse.com/pub/projects/security/csaf/suse-su-2019_0391-1.json"
			},
			{
				"category": "self",
				"summary": "URL for SUSE-SU-2019:0391-1",
				"url": "https://www.suse.com/support/update/announcement/2019/suse-su-20190391-1/"
			},
			{
				"category": "self",
				"summary": "E-Mail link for SUSE-SU-2019:0391-1",
				"url": "https://lists.suse.com/pipermail/sle-security-updates/2019-February/005119.html"
			},
			{
				"category": "self",
				"summary": "SUSE Bug 1120767",
				"url": "https://bugzilla.suse.com/1120767"
			},
			{
				"category": "self",
				"summary": "SUSE CVE CVE-2018-1000872 page",
				"url": "https://www.suse.com/security/cve/CVE-2018-1000872/"
			}
		],
		"title": "Security update for python-PyKMIP",
		"tracking": {
			"current_release_date": "2019-02-14T13:44:35Z",
			"generator": {
				"date": "2019-02-14T13:44:35Z",
				"engine": {
					"name": "cve-database.git:bin/generate-csaf.pl",
					"version": "1"
				}
			},
			"id": "SUSE-SU-2019:0391-1",
			"initial_release_date": "2019-02-14T13:44:35Z",
			"revision_history": [
				{
					"date": "2019-02-14T13:44:35Z",
					"number": "1",
					"summary": "Current version"
				}
			],
			"status": "final",
			"version": "1"
		}
	},
	"product_tree": {
		"branches": [
			{
				"branches": [
					{
						"branches": [
							{
								"category": "product_version",
								"name": "python-PyKMIP-0.6.0-3.3.1.noarch",
								"product": {
									"name": "python-PyKMIP-0.6.0-3.3.1.noarch",
									"product_id": "python-PyKMIP-0.6.0-3.3.1.noarch"
								}
							},
							{
								"category": "product_version",
								"name": "python3-PyKMIP-0.6.0-3.3.1.noarch",
								"product": {
									"name": "python3-PyKMIP-0.6.0-3.3.1.noarch",
									"product_id": "python3-PyKMIP-0.6.0-3.3.1.noarch"
								}
							}
						],
						"category": "architecture",
						"name": "noarch"
					},
					{
						"branches": [
							{
								"category": "product_name",
								"name": "HPE Helion OpenStack 8",
								"product": {
									"name": "HPE Helion OpenStack 8",
									"product_id": "HPE Helion OpenStack 8",
									"product_identification_helper": {
										"cpe": "cpe:/o:suse:hpe-helion-openstack:8"
									}
								}
							},
							{
								"category": "product_name",
								"name": "SUSE OpenStack Cloud 8",
								"product": {
									"name": "SUSE OpenStack Cloud 8",
									"product_id": "SUSE OpenStack Cloud 8",
									"product_identification_helper": {
										"cpe": "cpe:/o:suse:suse-openstack-cloud:8"
									}
								}
							},
							{
								"category": "product_name",
								"name": "SUSE OpenStack Cloud Crowbar 8",
								"product": {
									"name": "SUSE OpenStack Cloud Crowbar 8",
									"product_id": "SUSE OpenStack Cloud Crowbar 8",
									"product_identification_helper": {
										"cpe": "cpe:/o:suse:suse-openstack-cloud-crowbar:8"
									}
								}
							}
						],
						"category": "product_family",
						"name": "SUSE Linux Enterprise"
					}
				],
				"category": "vendor",
				"name": "SUSE"
			}
		],
		"relationships": [
			{
				"category": "default_component_of",
				"full_product_name": {
					"name": "python-PyKMIP-0.6.0-3.3.1.noarch as component of HPE Helion OpenStack 8",
					"product_id": "HPE Helion OpenStack 8:python-PyKMIP-0.6.0-3.3.1.noarch"
				},
				"product_reference": "python-PyKMIP-0.6.0-3.3.1.noarch",
				"relates_to_product_reference": "HPE Helion OpenStack 8"
			},
			{
				"category": "default_component_of",
				"full_product_name": {
					"name": "python-PyKMIP-0.6.0-3.3.1.noarch as component of SUSE OpenStack Cloud 8",
					"product_id": "SUSE OpenStack Cloud 8:python-PyKMIP-0.6.0-3.3.1.noarch"
				},
				"product_reference": "python-PyKMIP-0.6.0-3.3.1.noarch",
				"relates_to_product_reference": "SUSE OpenStack Cloud 8"
			},
			{
				"category": "default_component_of",
				"full_product_name": {
					"name": "python-PyKMIP-0.6.0-3.3.1.noarch as component of SUSE OpenStack Cloud Crowbar 8",
					"product_id": "SUSE OpenStack Cloud Crowbar 8:python-PyKMIP-0.6.0-3.3.1.noarch"
				},
				"product_reference": "python-PyKMIP-0.6.0-3.3.1.noarch",
				"relates_to_product_reference": "SUSE OpenStack Cloud Crowbar 8"
			}
		]
	},
	"vulnerabilities": [
		{
			"cve": "CVE-2018-1000872",
			"ids": [
				{
					"system_name": "SUSE CVE Page",
					"text": "https://www.suse.com/security/cve/CVE-2018-1000872"
				}
			],
			"notes": [
				{
					"category": "general",
					"text": "OpenKMIP PyKMIP version All versions before 0.8.0 contains a CWE 399: Resource Management Errors (similar issue to CVE-2015-5262) vulnerability in PyKMIP server that can result in DOS: the server can be made unavailable by one or more clients opening all of the available sockets. This attack appear to be exploitable via A client or clients open sockets with the server and then never close them. This vulnerability appears to have been fixed in 0.8.0.",
					"title": "CVE description"
				}
			],
			"product_status": {},
			"references": [
				{
					"category": "external",
					"summary": "CVE-2018-1000872",
					"url": "https://www.suse.com/security/cve/CVE-2018-1000872"
				},
				{
					"category": "external",
					"summary": "SUSE Bug 1120767 for CVE-2018-1000872",
					"url": "https://bugzilla.suse.com/1120767"
				}
			],
			"remediations": [
				{
					"category": "vendor_fix",
					"details": "To install this SUSE Security Update use the SUSE recommended installation methods like YaST online_update or \"zypper patch\".\n",
					"product_ids": [
						"HPE Helion OpenStack 8:python-PyKMIP-0.6.0-3.3.1.noarch",
						"SUSE OpenStack Cloud 8:python-PyKMIP-0.6.0-3.3.1.noarch",
						"SUSE OpenStack Cloud Crowbar 8:python-PyKMIP-0.6.0-3.3.1.noarch"
					]
				}
			],
			"scores": [
				{
					"cvss_v3": {
						"baseScore": 7.5,
						"baseSeverity": "HIGH",
						"vectorString": "CVSS:3.0/AV:N/AC:L/PR:N/UI:N/S:U/C:N/I:N/A:H",
						"version": "3.0"
					},
					"products": [
						"HPE Helion OpenStack 8:python-PyKMIP-0.6.0-3.3.1.noarch",
						"SUSE OpenStack Cloud 8:python-PyKMIP-0.6.0-3.3.1.noarch",
						"SUSE OpenStack Cloud Crowbar 8:python-PyKMIP-0.6.0-3.3.1.noarch"
					]
				}
			],
			"threats": [
				{
					"category": "impact",
					"date": "2019-02-14T13:44:35Z",
					"details": "important"
				}
			],
			"title": "CVE-2018-1000872"
		}
	]
}
